src/main/webapp/app/shared/table/GenomicIndicatorsTable.tsx

```diff
@@ -74,6 +76,9 @@ const GenomicIndicatorsTable = ({
         }
       }),
     );
+
+    fetchGenomicIndicators(genomicIndicatorsPath);
```

Comment:
I'm a bit confused to make this call when the callback in this useEffect also does the same thing